bench/Autofac.BenchmarkProfiling/Autofac.BenchmarkProfiling.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <PlatformTarget>x64</PlatformTarget>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="BenchmarkDotNet" Version="0.12.1" />
```

Comment:
Is this not sucked in as a transient dependency? Might make it easier to maintain if it's possible to use it by virtue of transience.

Reply:
Yeah, it will be; I think it was just the order I did things...